Front/addsavenote.h

```diff
@@ -10,25 +10,30 @@
 #include <memory.h>
 #include <QRegularExpression>
 #include <QStringConverter>
-
+#include "notesincategory.h"
 
 QT_BEGIN_NAMESPACE
 namespace Ui { class MainWindow; }
 QT_END_NAMESPACE
 
-class MainWindow : public QMainWindow
+class AddSaveNote : public QMainWindow
 {
     Q_OBJECT
 
 public:
-    MainWindow(QWidget *parent = nullptr);
-    ~MainWindow();
+    AddSaveNote(QWidget *parent = nullptr);
+    ~AddSaveNote();
 
 private slots:
-    void on_SaveNoteButton_clicked();
+    void on_ASNSaveNoteButton_clicked();
     void CreateJson(const QString &path);
 
+    void on_ASNCanselButton_clicked();
+
+    void on_ASNHomeButton_clicked();
+
 private:
     Ui::MainWindow *ui;
+    NotesInCategory *notesInCategory;
```

Comment:
This is a member of class, consider adding `m_` prefix

Reply:
Fixed